src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { ClearButton } from './ClearButton';
+import { Filter } from './Filter';
+import { TodoCount } from './TodoCount';
+
+export const Footer: React.FC = React.memo(() => {
```

Comment:
Consider adding an interface for the component's props to improve type-checking.

Reply:
I'm not sure that I understand correct. I don't have any props here